src/components/results/feedback/modal/index.js

```diff
@@ -0,0 +1,97 @@
+import {faTimes} from "@fortawesome/free-solid-svg-icons";
+import {feedbackModalShowState} from "lib/recoil";
+import {useSetRecoilState} from "recoil";
+import useTranslate from "lib/hooks/use-translate";
+import Icon from "components/common/icon";
+import useLoadedValue from "lib/hooks/use-loaded-value";
+import style from "./style.scss";
+import {feedbackSurveyQuery} from "../../../../lib/recoil/feedback";
+
+export default function FeedbackModal() {
+  const setShow = useSetRecoilState(feedbackModalShowState);
+  const feedbackSurvey = useLoadedValue(feedbackSurveyQuery);
+  const translate = useTranslate();
+
+  console.log("feedbackSurvey :>> ", feedbackSurvey);
+  if(!feedbackSurvey) { return null; }
+
+  const onCancel = () => {
+    setShow(false);
+  };
+
+  const onSubmit = (event) => {
+    event.preventDefault();
+    const response = new Map(new FormData(event.target).entries());
+    console.log("response :>> ", response);
+    // TODO send to server
+  };
+
+  const multipleChoice = (question) => (
+    <div key={question.id}>
+      <label htmlFor={question.id} className={style.question}>{question.text}</label>
+      <select form="form" className={style.dropdown} id={question.id} name={question.id} defaultValue="">
```

Comment:
We like to keep html attributes in alphabetical order, with the exception of the `key` used by react